src/index.mjs

```diff
@@ -99,9 +109,20 @@ if (paths.length === 0) {
     paths.push(".");
 }
 
-if (options.opt.locales) {
+if (options.opt.localefile) {
+    const json = json5.parse(fs.readFileSync(options.opt.localefile, "utf-8"));
```

Comment:
what is benefit to using json5 package instead of JSON.parse?
are there many differences between json and json5?

Reply:
It's much more forgiving. See https://json5.org/